tests/vmi_configuration_test.go

```diff
@@ -818,7 +820,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			})
 
 			It("[test_id:3074]should failed to start the VMI with wrong bus of input device", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				vmi := libvmi.NewCirros()
```

Comment:
Same as above

Reply:
I'm not sure why here, when this substitutes Cirros for Cirros, so no change?